homeworks/yana.haivoronska_Yana_Haivoronska/Game Project/style.css

```diff
@@ -0,0 +1,4 @@
+main {
+  width: 409px;
+  height: 575px;
```

Comment:
1) Please use max-width instead of width and remove fixing height.
2) Import font "Roboto" on your page from Google Fonts
3) Add styles for text, like font-family, font-size, color, etc.

Reply:
Good morning,  I hope it's better now .

Reply:
Good morning ,  I hope it's better now

Reply:
Good morning ,  I hope it's better now

Reply:
hi @YanaHaivoronska the issue with font-family is still unresolved. Please import the font on the page and then use it for title and text